Bottom: fd2be63602554b0fb762bd95a5a95fc3f594be74
Top:    91aede721681e4bab1b78147f554014d7f43d75d
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 16:52:24 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index fb79eb0..2139813 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -268,7 +268,7 @@ static int rpc_do_create_client(struct sockaddr *addr, struct conn_info *info, i
 		in4_raddr->sin_port = htons(info->port);
 		slen = sizeof(struct sockaddr_in);
 		/* Use rpcbind v2 for AF_INET */
-		if (info->program == rpc_prog)
+		if (info->program == rpcb_prog)
 			info->version = PMAPVERS;
 	} else if (addr->sa_family == AF_INET6) {
 		struct sockaddr_in6 *in6_raddr = (struct sockaddr_in6 *) addr;
